src/app/profile/resources/resources.component.html

```diff
@@ -0,0 +1,40 @@
+<div id="resources" class="container content">
```

Comment:
I think you may have accidentally added this files back in. The Resource component was deleted.